---
title: "erv_coverage"
author: "kira delmore"
date: "march 21, 2020"
output: 
  html_document: 
    keep_md: yes
---

```{r }
library(knitr)
dat <- read.csv("C:/Users/Kira/Dropbox/Projects_Desktop/Blackbird_Analysis/erv_coverage.csv")
```

#### average coverage for the 10 ERVs identified by censor  
```{r }
kable(colMeans(dat[,7:16]),col.names = c("avg_cov"), format = "markdown")
```

#### average coverage for housekeeping genes randomly selected from the genome
#### much higher then all ERVs
```{r }
kable(colMeans(dat[,20:24]),col.names = c("avg_cov"),  format = "markdown")
```

#### average genome coverage overall
#### again, much higher then all ERVs
```{r }
mean(dat$genome_cov)
```

#### coverage at the ERVs has nothing to do with phenotype
```{r }
kable(aggregate(dat[, 7:16], list(dat$pheno), mean), format = "markdown")
```

#### two of the snps with high fsts are in the second ERV identified by censor
```{r }
plot(dat$genome_cov,dat$erv_2,ylim=c(0,11))
```

#### there is a clear subset of birds at that ERV that have little to no coverage
#### this is unrelated to phenotype
```{r }
low_cov <- subset(dat,dat$erv_2<2)
kable(low_cov$pheno, format = "markdown")
```

#### it is like there are three variants here - deletion, 0 or 1 at the SNPs